src/DevTools/createTestEnv.tsx

```diff
@@ -147,13 +149,14 @@ class TestEnv<MutationNames extends string, TestPage extends RootTestPage> {
       })
 
     this.mutations.mockFetch.mockImplementation(wrappedFetchQuery)
+    this.mockQuery.mockImplementation(wrappedFetchQuery)
 
     // Switch on mutation/query when making requests to help make assertions
     // Seems we only make assertions about mutations right now
     const mockNetwork = Network.create((operation, variableValues) => {
       return operation.operationKind === "mutation"
         ? this.mutations.mockFetch(operation, variableValues)
-        : wrappedFetchQuery(operation, variableValues)
+        : this.mockQuery(operation, variableValues)
```

Comment:
@ds300  I wasn't sure how we should be testing "re-fetch" behavior so I just added the `mockQuery` variable to give us flexibility to update mocks in an `it` block. I wasn't sure if this is the intended approach and would like your eyes on this so we can make the testing foundation better while maintaining the original design and philosophy.

Reply:
Looks great! Thanks!